tests/src/rules/max-dependencies.js

```diff
@@ -91,7 +91,7 @@ ruleTester.run('max-dependencies', rule, {
   ],
 });
 
-context('TypeScript', { skip: semver.satisfies(eslintPkg.version, '>5.0.0') }, () => {
```

Comment:
`{ skip: ... }` isn't a thing, I think? We're passing the callback as the third argument -- if this is the [describe()/context() implementation](https://github.com/mochajs/mocha/blob/8421974e8d252e2ef11602d15bed178e4f5f4534/lib/interfaces/bdd.js#L41), then that argument is ignored?

Reply:
you're right; this is the way tape works, not BDD-style testing. Thanks for the fix.

Reply:
Ah, that makes sense. Thanks for explaining the confusion!

I'm done now, please merge when ready.

Reply:
Sorry, I take that back: there is a mistake. One sec ...